src/main/java/com/ibm/intoto/attestation/custom/resource/descriptors/file/FileResourcesDescriptor.java

```diff
@@ -30,28 +30,28 @@
 /**
  * A ResourceDescriptor type to encapsulate the WAR file created from a Maven project.
  */
-public class WarResourceDescriptor extends ResourceDescriptor {
+public class FileResourcesDescriptor extends ResourceDescriptor {
```

Comment:
```suggestion
public class FileResourceDescriptor extends ResourceDescriptor {
```
As this class is a type of `ResourceDescriptor`, I'd suggest the class should be called `FileResourceDescriptor` (no 's' on "Resource").